zebra/zebra_evpn_mh.c

```diff
@@ -3392,33 +3393,35 @@ DEFPY(zebra_evpn_es_id,
 			ret = zebra_evpn_es_type0_esi_update(zif, zero_esi);
 
 		if (ret == -1) {
-			vty_out(vty, "%%Failed to clear ES local id\n");
+			vty_out(vty,
+				"%% Failed to clear ES local id or ESI name\n");
 			return CMD_WARNING;
 		}
 	} else {
 		if (!zebra_evpn_is_if_es_capable(zif)) {
 			vty_out(vty,
-				"%%ESI cannot be associated with this interface type\n");
+				"%% ESI cannot be associated with this interface type\n");
 			return CMD_WARNING;
 		}
 
 		if (esi_str) {
 			if (!str_to_esi(esi_str, &esi)) {
-				vty_out(vty, "%% Malformed ESI\n");
+				vty_out(vty, "%% Malformed ESI name\n");
 				return CMD_WARNING;
 			}
 			ret = zebra_evpn_es_type0_esi_update(zif, &esi);
 		} else {
 			if (!es_lid) {
-				vty_out(vty, "%%Specify local ES ID\n");
+				vty_out(vty,
+					"%% Specify ES local id or ESI name\n");
```

Comment:
Can here be a name? Isn't the name already verified above?

Reply:
When run `evpn mh es-id` without both `local id` and `ESINAME`, you don't know the user wants to use `local id` or `ESINAME`.
   
So, the current warning is incomplete :
`` `
(config-if)# evpn mh es-id 
%Specify local ES ID
```